js/util/ajax.js

```diff
@@ -59,7 +61,11 @@ exports.getImage = function(url, callback) {
             (window.URL || window.webkitURL).revokeObjectURL(img.src);
         };
         var blob = new window.Blob([new Uint8Array(imgData)], { type: 'image/png' });
-        img.src = (window.URL || window.webkitURL).createObjectURL(blob);
+        if (imgData.byteLength) {
```

Comment:
> 10.2.1 200 OK

> The request has succeeded. The information returned with the response is dependent on the method used in the request, for example:
> 
> GET an entity corresponding to the requested resource is sent in the response;
> ...

My interpretation is that a 200 response must include a body. Per the spec, a trivial transparent PNG would be acceptable but the lack of a body would be unacceptable. In practice, a 200 response with no body is more likely a bug than intended behaviour.